kitchen-terraform.gemspec

```diff
@@ -74,7 +74,10 @@ require "kitchen/terraform/version.rb"
     )
 
   specification.add_runtime_dependency "kitchen-inspec", "~> 0.18"
-
+  specification.add_runtime_dependency "awspec",  "~> 1.2"
```

Comment:
I do not agree with adding awspec as a runtime dependency to the Kitchen-Terraform gem. Gems specific to testing particular providers should be managed in the Gemfiles of the projects which are testing against those providers.

Reply:
Understood.  Thinking of changing the dockerfile to have an ONBUILD instruction to create a gem locally as you suggest in your examples with awspec.